test/Microsoft.TestPlatform.AcceptanceTests/Extension/RunnnerInfo.cs

```diff
@@ -22,7 +35,7 @@ public record RunnerInfo(string RunnerFramework, string TargetFramework, string
     /// <summary>
     /// Is running via .NET Framework vstest.console?
     /// </summary>
-    public bool IsNetFrameworkRunner => RunnerFramework.StartsWith("net4", StringComparison.InvariantCultureIgnoreCase);
+    public bool IsNetFrameworkRunner => RunnerFramework!.StartsWith("net4", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Shall we go in the "safe way" and do

```suggestion
    public bool IsNetFrameworkRunner => RunnerFramework?.StartsWith("net4", StringComparison.InvariantCultureIgnoreCase) ?? false;
```

or do we prefer to have a throw if that's not set.

Reply:
Yes let it throw.